bin/jade.js

```diff
@@ -144,6 +130,51 @@ if (files.length) {
 }
 
 /**
+ * Watch for changes on path
+ * Renders base if specified, otherwise renders path
+ * If watched path was moved or removed, unwatches the path
+ */
+function watchFile(path, base) {
+  path = normalize(path);
```

Comment:
We should use `resolve` rather than `normalize` here.  Probably doesn't make any difference, but what we want is the full absolute path.